dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -65,12 +65,18 @@ def get_ai_evaluations
     return head :not_found unless student
     return head :forbidden unless can?(:manage, student)
 
-    learning_goals = LearningGoal.where(rubric_id: permitted_params[:id])
-    # Get the most recent AI evaluation for each learning goal
-    learning_goal_ai_evaluations =
-      LearningGoalAiEvaluation.where(user_id: permitted_params[:student_id], learning_goal_id: learning_goals.pluck(:id)).
-        group_by(&:learning_goal_id).
-        map {|_, eval_list| eval_list.max_by(&:updated_at)}
+    learning_goal_ids = LearningGoal.where(rubric_id: permitted_params[:id]).select(:id)
```

Comment:
I replaced `pluck` (which I copied from a different function in this controller pulling out learning goal ids) with `select` to avoid an awkward query.